client/rest-high-level/src/test/java/org/elasticsearch/client/IndicesClientIT.java

```diff
@@ -22,12 +22,55 @@
 import org.elasticsearch.ElasticsearchException;
 import org.elasticsearch.action.admin.indices.delete.DeleteIndexRequest;
 import org.elasticsearch.action.admin.indices.delete.DeleteIndexResponse;
+import org.elasticsearch.action.admin.indices.exists.indices.IndicesExistsRequest;
+import org.elasticsearch.action.admin.indices.exists.indices.IndicesExistsResponse;
 import org.elasticsearch.rest.RestStatus;
 
 import java.io.IOException;
 
 public class IndicesClientIT extends ESRestHighLevelClientTestCase {
 
+    public void testIndexExists_IndexPresent() throws IOException {
```

Comment:
I would prefer not using underscore as part of method names if possible. can you remove this one and the following ones please?

Reply:
Done!